src/overlays/actors/ovl_En_Clear_Tag/z_en_clear_tag.c

```diff
@@ -61,25 +76,999 @@ static ColliderCylinderInit D_809D5C40 = {
+            case CLEAR_TAG_STATE_TARGET_LOCKED:
+            case CLEAR_TAG_STATE_FLYING:
+                // Check if the Arwing should crash.
+                if (this->collider.base.acFlags & 2 &&
+                    (this->collider.base.acFlags &= ~2, this->crashingTimer = 20,
+                     Actor_SetColorFilter(&this->actor, 0x4000, 255, 0, 5), this->acceleration.x = Rand_CenteredFloat(15.0f),
+                     this->acceleration.y = Rand_CenteredFloat(15.0f), this->acceleration.z = Rand_CenteredFloat(15.0f),
+                     Audio_PlayActorSound2(&this->actor, NA_SE_EN_FANTOM_THUNDER_GND), this->actor.colChkInfo.health--,
+                     (s8)this->actor.colChkInfo.health <= 0)) {
+                    this->state = CLEAR_TAG_STATE_CRASHING;
+                    this->actor.velocity.y = 0.0f;
+                } else {
+                    Actor_SetScale(&this->actor, 0.2f);
+                    this->actor.speedXZ = 7.0f;
+
+                    // Update the state based on the work timer.
+                    if (this->work[0] == 0) {
+                        if (this->work[1] == 0) {
+                            this->state = CLEAR_TAG_STATE_TARGET_LOCKED;
+                            this->work[0] = 300;
+                        } else {
+                            this->state = CLEAR_TAG_STATE_FLYING;
+                            this->work[0] = ((s16)Rand_ZeroFloat(50.0f)) + 20;
+
+                            if (this->actor.params == CLEAR_TAG_ARWING) {
+                                // Set the Arwing to fly in a circle around the player.
+                                targetCircleX = Math_SinS(player->actor.shape.rot.y) * 400.0f;
+                                targetCircleZ = Math_CosS(player->actor.shape.rot.y) * 400.0f;
+                                this->targetPosition.x =
+                                    Rand_CenteredFloat(700.0f) + (player->actor.world.pos.x + targetCircleX);
+                                this->targetPosition.y = Rand_ZeroFloat(200.0f) + player->actor.world.pos.y + 150.0f;
+                                this->targetPosition.z =
+                                    Rand_CenteredFloat(700.0f) + (player->actor.world.pos.z + targetCircleZ);
+                            } else {
+                                // Set the Arwing to fly to a random position.
+                                this->targetPosition.x = Rand_CenteredFloat(700.0f);
+                                this->targetPosition.y = Rand_ZeroFloat(200.0f) + 150.0f;
+                                this->targetPosition.z = Rand_CenteredFloat(700.0f);
+                            }
+                        }
+
+                        this->targetDirection.z = 0.0f;
+                        this->targetDirection.y = 0.0f;
+                        this->targetDirection.x = 0.0f;
+                    }
+
+                    rotationScale = 10;
+                    xRotationTarget = 2048;
```

Comment:
```suggestion
                    xRotationTarget = 0x800;
```
Power of 2 angles should *definitely* be in hex.